opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import typing
+from enum import Enum
+
+from .. import Span, SpanProcessor
+
+
+class ExportResult(Enum):
+    SUCCESS = 0
+    FAILED_RETRYABLE = 1
+    FAILED_NONE_RETRYABLE = 2
+
+
+class Exporter:
```

Comment:
How about making `Exporter` a child class of `SpanProcessor`?

Reply:
Also IIRC, processors will be allowed to write/update the `Span`, while exporters are expected to get read-only `Span` data.

(We will be using `Span`s for now, but this might change in the future, of course).